index.js

```diff
@@ -12,6 +12,63 @@ function query(query, format = "CSV") {
   return chdbNode.Query(query, format);
 }
 
+// Standalone exported query function
+function queryBuffer(query, format = "CSV") {
+  if (!query) {
+    return "";
+  }
+  return chdbNode.QueryBuffer(query, format);
+}
+
+class LocalChDB {
```

Comment:
refer to the libchdb function called here, Connect is a better name than LocalChDB